src/components/DefaultRedirectionSnackbar/helpers.js

```diff
@@ -0,0 +1,84 @@
+import { deconstructRedirectLink, hasQueryBeenLoaded } from 'cozy-client'
+
+export const NAV_APP_SHOW_THRESHOLD = 3
+
+export const HOME_DEFAULT_REDIRECTION = 'home/'
+
+export const incrementNavAppShowCount = async (
+  client,
+  instanceSettingsResult
+) => {
+  const { nav_app_show_count } = instanceSettingsResult.data.attributes
+
+  return await client.getStackClient().fetchJSON('PUT', '/settings/instance', {
+    data: {
+      ...instanceSettingsResult.data,
+      attributes: {
+        ...instanceSettingsResult.data.attributes,
+        nav_app_show_count: nav_app_show_count ? nav_app_show_count + 1 : 1
+      }
+    }
+  })
+}
+
+export const disableDefaultRedirectionSnackbar = async (
+  client,
+  instanceSettingsResult
+) => {
+  return await client.getStackClient().fetchJSON('PUT', '/settings/instance', {
+    data: {
+      ...instanceSettingsResult.data,
+      attributes: {
+        ...instanceSettingsResult.data.attributes,
+        default_redirection_snackbar_disabled: true
+      }
+    }
+  })
```

Comment:
Why not using client.save() instead of a call to fetchJSON? With your current call, we're not updating the cozy-client internal store.

Reply:
I updated with client.save(). It works without creating a new document without _rev, maybe because it's _io.cozy.settings.instance_ ?